src/global_timestamp_reader.cpp

```diff
@@ -111,9 +112,26 @@ namespace librealsense
 
     void time_diff_keeper::start()
     {
+        _users_count++;
```

Comment:
Start/Stop shall be muted-protected as the invocation can be done from multiple threads.